composer.json

```diff
@@ -36,7 +36,7 @@
         "symfony/dependency-injection": "^6.1.3",
         "symfony/finder": "^6.1.3",
         "symfony/string": "^6.1.4",
-        "symplify/autowire-array-parameter": "^11.1.10",
+        "symplify/autowire-array-parameter": "dev-main as 11.1.15",
```

Comment:
@TomasVotruba this is temporary needed while wait for next symplify/autowire-array-parameter release, ref https://github.com/symplify/symplify/pull/4458

Reply:
Release is on the way :+1:  ETA 5 mins

Reply:
Thank you, rolled back https://github.com/rectorphp/rector-src/pull/3017/commits/6a4947446130208f54b9b688f10bc0a6a9527ad3